docs/reference/units.ipynb

```diff
@@ -448,6 +471,192 @@
    "source": [
     "sc.to_unit(1.0 * sc.Unit('s'), 'm')"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Unit Aliases\n",
+    "\n",
+    "It is possible to define custom aliases for units.\n",
+    "This can be used to\n",
+    "\n",
+    "- guide string formatting to prefer a certain unit, e.g. angstrom over nm\n",
+    "- define domain specific units that can be expressed in terms of other units to\n",
+    "  - guide string formatting\n",
+    "  - construct units from strings with custom names\n",
+    "  \n",
+    "## Prioritizing Units in String Formatting\n",
+    "\n",
+    "When dealing with crystals or molecules, it is often convenient to use angstrom as a unit.\n",
+    "But by default, string formatting tends to prefer different bases in composite units.\n",
+    "For example"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sc.Unit('us/angstrom**2')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This result is not very useful.\n",
+    "We can make the formatter prefer angstrom by defining it as an alias of itself (or alternatively of `'10^-10 m'`):"
```

Comment:
How will it format 'meter' in this case?

Reply:
Still as `m`:
```.py
import scipp as sc
sc.units.aliases['angstrom'] = 'angstrom'
for p in range(12):
    print(sc.Unit(f'{10**-p}*m'))
```
```
m
dm
cm
mm
0.0001m
1e-05m
µm
9.99999999999999955e-08m
1e-08m
nm
angstrom
9.99999999999999939e-12m
```
This case is unaffected because LLNL/Units already knows about angstrom. The alias is more useful here:
```.py
import scipp as sc
sc.units.aliases['angstrom'] = 'angstrom'
for p in range(12):
    print(sc.Unit(f'{10**(2*p)}*us/m^2'))
```
left without alias, right with alias:
```
s/km^2          s/km^2
s/hectare       s/hectare
s/are           s/are
s/m^2           s/m^2
100T/V          as/angstrom^2
s/cm^2          s/cm^2
MT/V            MT/V
100000000T/V    ps/angstrom^2
10000000000T/V  10000000000T/V
TT/V            TT/V
1e+20s/km^2     µs/angstrom^2
1e+19ms/m^2     1e+19ms/m^2
```